demo/api/schema.gql

```diff
@@ -1413,11 +1417,13 @@ input ProductUpdateInput {
 input ProductCategoryInput {
   title: String!
   slug: String!
+  position: Float!
 }
 
 input ProductCategoryUpdateInput {
   title: String
   slug: String
+  position: Float
```

Comment:
This should be Int

Reply:
this results from generated code. entity should to be correct: [@Field(() => Int)](https://github.com/vivid-planet/comet/pull/2216/files#diff-71dbe8285067476a8ecc357b91ac5cf5ebb3c7c41742c5cfc914ce3787fc33a5R27)

this is the generated [input](https://github.com/vivid-planet/comet/pull/2216/files#diff-a8a58070bc828c4058a36860cc7466127c2f62d37e4826f7d124478906f67b60R22)

Reply:
you mean this shows a bug in the api-generator?

Reply:
yes, exactly. didn't even noticed it myself before you mentioned it.

Reply:
> how did you create the mutation? manually?

do you mean `updateProductCategoryPosition`? this is created manually for demo purposes. everything else was just running api-generator.

Reply:
ah, sorry I meant the migration

Reply:
ah, yes. I created it manually because auto-generated migration fails if product-category-table has some rows.

Reply:
please don't manually generate. you can add a truncate/delete before the alter table, see existing migrations

Reply:
Ok, I generated the file but changed the content. I called `mikro-orm:migration:generate` again and it worked (the change was correctly detected and not generated again). It seems this migration script is ok. Or do you mean some other problem?

Reply:
> I called mikro-orm:migration:generate again and it worked (the change was correctly detected and not generated again).

This can't be true to my understanding:

- you are using `@Property() position: number`
- that will create a float by default
- you wrote the migration script manually that creates an int
- if you run generation script again you should get a migration that changes the type from int to float

Reply:
I can't explain why but I tried it again, reset product-category.position to 
```
@Property()
@Field(() => Int)
position: number;
```
deleting position column, deleting migration, restarting api, running fixtures, generating migration.

result (with a lot of other changes...)
`this.addSql('alter table "ProductCategory" add column "position" int not null;');`
I even checked out comet in another folder, same result...

Reply:
ok

(I have no idea)

Reply:
the bug with generated input-type is fixed in #2225

Reply:
I mistakenly removed the migration, so I added it again.